content/en/getting_started/agent/_index.md

```diff
@@ -12,90 +12,219 @@ further_reading:
+
+### Data Collected
+
+#### Agent metrics
+
+| Metric                           | Description                                                                                                          |
+| -------------------------------- | -------------------------------------------------------------------------------------------------------------------- |
+| **datadog.agent.python.version** | Shows a value of `1` if the Agent is reporting to Datadog. The metric is tagged with the `python_version`. |
+| **datadog.agent.running**        | Shows a value of `1` if the Agent is reporting to Datadog.                                                 |
+| **datadog.agent.started**        | A count sent with a value of `1` when the Agent starts (available in v6.12+).                                        |
+
+See the [Agent Metrics][3] integration for a full list of Agent metrics.
+
+#### Checks
+
+Depending on your platform, the Agent has several core checks enabled by default that collect metrics.
+
+| Check       | Metrics       | Platforms          |
+| ----------- | ------------- | ------------------ |
+| CPU         | [System][4]  | All                |
+| Disk        | [Disk][5]    | All                |
+| IO          | [System][4]  | All                |
+| Memory      | [System][4]  | All                |
+| Network     | [Network][6] | All                |
+| NTP         | [NTP][7]     | All                |
+| Uptime      | [System][4]  | All                |
+| File Handle | [System][4]  | All except Mac     |
+| Load        | [System][4]  | All except Windows |
+| Docker      | [Docker][8]  | Docker             |
+| Winproc     | [System][4]  | Windows            |
+
+To collect metrics from other technologies, see the [Integrations][9] page.
+
+## Containerized Environments
+
+There are a few differences in the way the Agent runs in a containerized environment.
+
+1. The Agent configuration options are passed in with [Environment Variables][10], for example:
+    - `DD_API_KEY` for the Datadog API key
+    - `DD_SITE` for the Datadog site
+
+2. Integrations are automatically identified through Datadog's Autodiscovery feature. See [Basic Agent Autodiscovery][11] to learn more.
+
+See the [Docker Agent][12] or [Kubernetes][13] for a walkthrough on running the Agent in a containerized environment.
+
+## Why Should I Install the Agent?
+
+The Agent needs to be installed to pull data from any one of the many Agent based Integraions. The Agent is not necessarily required to forward data to the Datadog Platform, for example, you can send Logs and Metrics through the Datadog API. However, the Agent is the recommended method to forward your data to the Datadog Platform. 
+
+The Agent samples host data  every 15 seconds to provide a more accurate understanding of what is happening across your environments. Additionally, it comes with over 50 default metrics to provide greater insight on system level data.
```

Comment:
```suggestion
The Agent collects host data every 15 seconds to provide an accurate understanding of what is happening across your environments. Additionally, it comes with over 50 default metrics to provide greater insight on system level data.
```

This is a confusing (kinda inaccurate) sentence. One agent (singular) doesn't provide cross-environment information. And the agent doesn't "come with" metrics -- it does collect some system level metrics by default, though.